tests/src/test_artifacts/reentrancy_attacker_contract/src/main.sw

```diff
@@ -0,0 +1,75 @@
+contract;
+
+use std::{auth::*, call_frames::contract_id};
+
+use reentrancy_target_abi::Target;
+use reentrancy_attacker_abi::Attacker;
+
+// Return the sender as a ContractId or panic:
+fn get_msg_sender_id_or_panic(result: Result<Identity, AuthError>) -> ContractId {
+    match result {
+        Result::Ok(s) => {
+            match s {
+                Identity::ContractId(v) => v,
+                _ => revert(0),
+            }
+        },
+        _ => {
+            revert(0);
+        },
+    }
+}
+
+impl Attacker for Contract {
+    fn launch_attack(target: ContractId) -> bool {
+        let id = target.value;
+        let target = abi(Target, id);
```

Comment:
```suggestion
        let target = abi(Target, target.value);
```

No reason to have an additional variable here

Reply:
Removed single-use variables